run.py

```diff
@@ -509,8 +510,9 @@ def run(self):
         sys.stdout.write("===> %s: " % os.path.basename(self.directory))
         sys.stdout.flush()
 
-        self.check_requires()
-        self.check_skip()
+        if not self.force:
+            self.check_requires()
+            self.check_skip()
```

Comment:
Just to be clear if there is `force` argument then we're not supposed to check for `requires` or `skip`?

Reply:
Yeah, I think it should. Sometimes you just need to force a test to run without modifying it.